<html>
	<head>
		<title>Stephanie Yan</title>
		<meta charset="utf-8" />
		<link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/custom.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="#" class="image avatar"><img src="images/avatar.jpg" alt="Link to work" /></a>
					<h1> Stephanie Yan </h1>
						<p>PhD candidate studying human genetics, genomics, & evolution</p>
						<ul>
							<li><a href="#research"> Research Projects </a></li>
							<li><a href="#service"> Academic Service </a></li>
							<li><a href="#teaching"> Teaching </a></li>
							<li><a href="#personal"> Personal Projects </a></li>
							<li><a href="pages/StephanieMYan_CV.pdf"> Download CV </a></li>
						</ul>
				</div>
			</header>

		<!-- Main -->
			<div id="main">

				<!-- Bio -->
					<section id="bio">
						<p>I’m a PhD candidate in the <a href="https://cmdb.jhu.edu/"> Cell, Molecular, Developmental Biology, and Biophysics</a> (CMDB) program at Johns Hopkins University, in the lab of <a href="https://mccoy-lab.org/"> Rajiv McCoy</a>. I study the impacts of large genetic variants, called <i>structural variants</i>, on human evolution.</p>
						<p>Before starting my PhD, I graduated from Cornell University, where I studied membrane trafficking in yeast in the lab of <a href="https://emr.wicmb.cornell.edu/"> Scott Emr</a>.</p>
					</section>

				<!-- Research -->
					<section id="research">
						<h2>Research Projects</h2>

						<div class="row">
							<article class="col-6 col-12-xsmall work-item">
								<a href="https://elifesciences.org/articles/67615" target="_blank"><img src="images/thumbs/sv_selection.png" alt="Recombination in an adaptive immunoglobulin haplotype" /></a>
								<h3>Local adaptation at human structural variant loci</h3>
								<p>Large genomic rearrangements (insertions, deletions, etc.), called structural variants (SVs), are an understudied form of genetic variation that can have dramatic functional impacts. These variants are hard to discover with normal sequencing methods, and instead require expensive and low-throughput long-read sequencing to study comprehensively. We used a hybrid approach to study SVs on a population-wide scale: SV discovery with long-read sequencing + computational genotyping in a large dataset of diverse human individuals. We found evidence of SVs under positive selection in specific populations, including a sequence in the immunoglobulin locus that was inherited into humans from Neanderthals.</p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="https://www.biorxiv.org/content/10.1101/2021.07.12.452063v1" target="_blank"><img src="images/thumbs/t2t_variants.png" alt="Frequency-differentiated variants in T2T-CHM13 novel regions" /></a>
								<h3>Characterizing a complete human reference genome</h3>
								<p>When the human genome was first sequenced in 2001, 8% of its most complex and repetitive regions were inaccessible to current sequencing technologies, and consequently remained incomplete for the next two decades. In 2021, the Telomere-to-Telomere Consortium used long-read sequencing to assemble the first ever complete human reference genome, <a href="https://www.biorxiv.org/content/10.1101/2021.05.26.445798v1"> T2T-CHM13</a>. Within this collaborative project, I analyzed the ancestry and Neanderthal-introgressed regions of the new reference. I also studied regions of the genome that were revealed for the first time by the T2T-CHM13 assembly, to understand how variants in these regions may affect human health. </p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<a href="https://www.sciencedirect.com/science/article/abs/pii/S0959437X20300654" target="_blank"><img src="images/thumbs/gd_review.png" alt="Prediction of archaic gene expression by Colbran et al. (2019)" /></a>
								<h3>Review: Studying archaic hominin gene expression</h3>
								<p>Recent research on Neanderthals and Denisovans, the closest evolutionary relatives to modern humans, has begun exploring patterns of gene expression to better understand how these archaic hominins may have been different from modern humans. Studying gene expression means working on RNA, which is too fragile to survive the tens of thousands of years that separate us from archaic hominins. As a result, researchers have developed many creative approaches for inferring archaic gene expression from DNA sequencing data. We summarized these methods in this review.</p>
							</article>
						</div>

					</section>

				<!-- Service -->
					<section id="service">
						<h2>Academic Service</h2>

						<div class='year'>
							<h3> symBIOsis </h3>
							<hr>
						</div>

						<p><a href="https://sites.krieger.jhu.edu/symbiosis/"> symBIOsis</a> is a graduate student support and mentorship group in the Department of Biology, founded in 2020 by me and other members of <a href="https://biominds.jhu.edu/"> MInDS</a>. Our goal is to improve the CMDB student experience by advising current students on graduate life, providing them with resources for navigating the program requirements, and helping them acclimate to the department. Among other things, we organize:</p>
						<ul>
							<li>Practice rotation talks for first years</li>
							<li>Mock qualifying exams for second years</li>
							<li>Advising meetings on topics like choosing thesis labs and writing grant applications</li>
							<li>Social events (happy hours, game nights, movie screenings)</li>
							<li>The BioBuddies program, which pairs incoming CMDB first years with upper-year mentors</li>
						</ul>

						<div class='year'>
							<h3> BioReps </h3>
							<hr>
						</div>

						<p>The BioReps are liasons between their graduate cohort and the CMDB program directors and administration. They’re also involved in organizing recruitment for prospective graduate students every year. Initiatives I’ve been part of as a BioRep include:</p>
						<ul>
							<li>Administering a survey on department climate issues</li>
							<li>Trying to implement a teaching feedback system for graduate TAs</li>
							<li>Advocating for student concerns and organizing department town halls during the COVID-19 pandemic</li>
						</ul>

					</section>

				<!-- Teaching -->
					<section id="teaching">
						<h2>Teaching</h2>

						<div class='year'>
							<h3> TAing </h3>
							<hr>
						</div>

						<p>I'm a teaching assistant for the CMDB program's one-week <a href="http://bxlab.github.io/cmdb-bootcamp/"> Quantitative Biology Bootcamp</a> (<b>AS.020.607</b>), and also TA'd its <a href="http://bxlab.github.io/cmdb-lab/"> Quantitative Biology Lab</a> course (<b>AS.020.617</b>) last year. These courses introduce new PhD students — most of whom have no coding experience — to computational biology. They learn Python and bash, and apply their skills to essential biological analyses like genome assembly, sequence alignment, GWAS, and RNA-seq. Over the past two years, my fellow TAs and I have also been instrumental in converting these hands-on, coding-based courses, which normally rely on in-person debugging and assistance, into online and hybrid formats for teaching during the pandemic.</p>
						<p>In the 2019-2020 academic year, I was a teaching assistant for JHU’s undergraduate Developmental Genetics Lab (<b>AS.020.340</b>), where I led students through designing and generating a CRISPR strain in <i>C. elegans</i>.</p>

						<div class='year'>
							<h3> Technology Fellowship </h3>
							<hr>
						</div>

						<p>This year, I was awarded a <a href="https://cer.jhu.edu/programs-and-services/techfellows"> Technology Fellowship</a> from the Johns Hopkins Center for Educational Resources. As a Technology Fellow, I'm partnering with Rajiv and Kate Weaver, another PhD student in the McCoy lab, to develop open-source modules for introducing undergraduates to computational genetics. These modules, formatted as interactive R Markdown files, adapt data from recent papers to introduce key human genetics concepts, teach the basics of coding in R, and allow for hands-on learning through built-in coding exercises and homework. These modules will be publicly available on Github for use in undergraduate courses or personal learning.</p>

					</section>


				<!-- Personal -->
					<section id="personal">
						<h2>Personal Projects</h2>

						<div class="row">
							<article class="col-6 col-12-xsmall work-item">
								<img src="images/thumbs/pokemon_curry.png" alt="Homemade 'Decorative Curry'" /></a>
								<h3>Pokémon Sword and Shield Curry</h3>
								<p>From May 2020 to May 2021, I painstakingly recreated all 26 varieties of curry from Pokémon Sword and Shield's <a href="https://www.serebii.net/swordshield/currydex.shtml"> Curry Dex</a>. A page with recipes and definitive curry rankings is coming soon! (Spoiler: The best ones were Apple Curry, Coconut Curry, and one variant of Egg Curry.)</p>
							</article>
						</div>

					</section>

				</section>


		<!-- Footer -->
			<footer id="footer">
				<div class="inner">
					<ul class="icons">
						<li><a href="https://scholar.google.com/citations?&user=Nudz5VkAAAAJ&sortby=pubdate" class="icon fa-graduation-cap" target="_blank"><span class="label">Google Scholar</span></a></li>
						<li><a href="http://twitter.com/smyan__" class="icon fa-twitter" target="_blank"><span class="label">Twitter</span></a></li>
						<li><a href="http://github.com/stephaniemyan" class="icon fa-github" target="_blank"><span class="label">Github</span></a></li>
					</ul>
					<ul class="copyright">
						<li>&copy; Stephanie Yan</li><li>Template: <a href="http://html5up.net">HTML5 UP</a>; <a href="https://bzjin.github.io/">Beatrice Jin</a></li>
					</ul>
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.poptrox.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
